bin/init.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+WD=$(dirname $0)
+WD=$(cd $WD; pwd)
+ROOT=$(dirname $WD)
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+# Ensure expected GOPATH setup
+if [ $ROOT != "${GOPATH-$HOME/go}/src/istio.io/istio" ]; then
+       echo "Istio not found in GOPATH/src/istio.io/"
+       exit 1
+fi
+
+function linkpkg() {
+  root=$1
+  shift
+  genfiles=$1
+  shift
+  comp=$1
+
+  pushd $genfiles
+  for fl in $(find $comp -type f);do
+    ln -sf ${genfiles}/$fl ${root}/$(dirname $fl)
+  done
+
+  popd
+}
+
+# This step is to fetch resources and create genfiles
+bazel build //...
+
+source "${ROOT}/bin/use_bazel_go.sh"
+
+# Clean up vendor dir
+rm -rf ${ROOT}/vendor
+mkdir -p ${ROOT}/vendor
+
+# Vendorize bazel dependencies
+${ROOT}/mixer/bin/bazel_to_go.py ${ROOT}
```

Comment:
you also need to run the bin/init.sh equivalent in pilot.

Reply:
Mixer bazel to py is. Super set of others.
Just with this, pilot mixer security and broker all ‘go build’ fine.

Reply:
but that path fix script below is only fixing mixer paths.. What about pilot paths?